### ISSUE TITLE
debug hook doesn't log error on error hook

### ISSUE THREAD

**Issue by bertho-zero** [feathersjs/feathers-hooks-common#152]
In my service i have:
```js
service.hooks({
  error: {
    all: hooks.debug()
  }
})
```

Which displays:
```
* 
type:error, method: create
data: { email: 'kevin.bertho@gmail.com',
  password: 'test',
  password_confirmation: 'test' }
query: {}
```

But my `hook.error` is:
```
{ BadRequest: Validation failed
    at BadRequest.ExtendableBuiltin (/home/bertho/dev/location-booking/node_modules/feathers-errors/lib/index.js:21:28)
    at BadRequest.FeathersError (/home/bertho/dev/location-booking/node_modules/feathers-errors/lib/index.js:96:116)
    at new BadRequest (/home/bertho/dev/location-booking/node_modules/feathers-errors/lib/index.js:137:105)
    at /home/bertho/dev/location-booking/api/hooks/validateHook.js:10:15
  type: 'FeathersError',
  name: 'BadRequest',
  code: 400,
  className: 'bad-request',
  data: { email: 'Already exist' },
  errors: {},
  hook: 
   { data: 
      { email: 'kevin.bertho@gmail.com',
        password: 'test',
        password_confirmation: 'test' },
     params: { query: {}, provider: 'socketio' },
     callback: [Function: callbackWrapper],
     method: 'create',
     type: 'before',
     app: { ... },
     service: { ... },
     path: 'users' } }
```

**Comment by eddyystop** [comment]
Yup, `debug` needs some love. Thanks for pointing this out.

**Comment by eddyystop** [comment]
Fixed in https://github.com/feathersjs/feathers-hooks-common/pull/167 will be published in v3.2.1.
